CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.11.0] - 2012-10-18
```

Comment:
Usually square brackets are for URLs, like this: https://github.com/lightstep/otel-launcher-go/blob/main/CHANGELOG.md?plain=1#L11 

Is the plan to add links here later? I think links to the tags would be good...for unreleased, we can do a compare from the latest version:
https://github.com/lightstep/telemetry-generator/compare/v0.11.0...HEAD
but then it will need to be updated with each release. 😬

Reply:
Ahh I see what you mean. I added the URL to the Version Number to be a link, to fit with the convention in otel-launcher-go  https://github.com/lightstep/telemetry-generator/pull/24/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR9